docs/adguard-for-android/solving-problems/outbound-proxy.md

```diff
@@ -109,8 +117,8 @@ Proxy type | *SOCKS5*
 Proxy host | *127.0.0.1*
 Proxy port | *1080*
 
-3. Tap **Save and select**
+5. Enable the main Proxy switch and AdGuard protection to route your device's traffic through the proxy.
```

Comment:
```
1. ...
2. ...
5. ...
```

why not just use lazy numbering?
https://google.github.io/styleguide/docguide/style.html#lists

Reply:
We should definitely do that